src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.html

```diff
@@ -29,8 +29,8 @@
       <alg-boolean-section
         i18n-title title="Can Enter"
         icon="fa fa-door-open"
-        [defaultValue]="this.permissions.can_enter_from"
-        (change)="this.permissions.can_enter_from = $event"
+        [defaultValue]="this.permissions.can_enter_from !== null"
+        (change)="this.permissions.can_enter_from = null"
       >
         <ng-template #label i18n>{{targetTypeString}} may enter this item (a contest or time-limited chapter)</ng-template>
       </alg-boolean-section>
```

Comment:
Should I handle the Can Enter section with the date picker like the design? I didn't for now because the date picker component is not part of the project now and I think it should be added in a different PR
![image](https://user-images.githubusercontent.com/31007874/104172818-25c26a80-5405-11eb-8f0e-01b1eb37a72c.png)

Reply:
I think we discussed it at the previous PR already: do not handle "can enter" for now.